### ISSUE TITLE
Bitmap fonts (PCF or OTB) don't render in Winamp playlist

### ISSUE THREAD

**Issue by Kodden585** [audacious-media-player/audacious#1505]
**Describe the bug**
As of a version or two ago, bitmap fonts don't render in the playlist anymore. In this case it specifically refers to the font Terminus, it's installed in both otb and pcf-format, and I'm not sure which one is loaded by Audacious.

**Steps to reproduce**
Set Audacious to Winamp mode, load any Winamp skin, and set the Playlist font to a bitmap font, or at least Terminus

**Expected behavior**
I expected the font to render as it should, as happened before.

**Additional information**
* Audacious version 4.4.2, Winamp interface
* Artix Linux
* XFCE
* 
![20241104161740](https://github.com/user-attachments/assets/b86de1f9-bcbc-4043-acc4-8473db037db4)
```
[user@workstation sidecar]$ LANG=C audacious -V
INFO ../audacious-4.4.2/src/audacious/dbus-server.cc:1132 [name_acquired]: Owned D-Bus name (org.atheme.audacious) on session bus.
INFO ../audacious-4.4.2/src/audacious/main.cc:399 [main]: No remote session; starting up.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dcc525c0> buffering enabled for /home/user/.config/audacious/config
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dcc525c0> open (mode r) /home/user/.config/audacious/config
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Transport/mms.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Transport/gio.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Transport/neon.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/asx.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/pls.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/cue.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/m3u.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/asx3.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/xspf.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Container/audpl.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/sndfile.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/console.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/xsf.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/vtx.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/ffaudio.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/flacng.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/modplug.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/sid.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/madplug.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/tonegen.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/cdaudio-ng.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/vorbis.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/aac-raw.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/metronom.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/openmpt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/amidi-plug.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/opus.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/psf2.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Input/wavpack.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/sdlout.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/alsa.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/oss4.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/pipewire.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/filewriter.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/jack-ng.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Output/pulse_audio.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/stereo.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/bitcrusher.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/crossfade.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/background_music.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/speed-pitch.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/echo.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/sox-resampler.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/resample.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/voice_removal.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/silence-removal.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/compressor.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/mixer.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/bs2b.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Effect/crystalizer.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/cd-menu-items.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/playlist-manager-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/delete-files.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/skins-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/song-info-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/scrobbler.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/statusicon-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/song_change.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/lyrics-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/mpris2.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/search-tool-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/qtui.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/lirc.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/albumart-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/notify.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/General/qthotkey.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Visualization/gl-spectrum-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Visualization/qt-spectrum.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Visualization/blur_scope-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-registry.cc:556 [plugin_register]: Register plugin: /usr/lib/audacious/Visualization/vumeter-qt.so
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:133 [start_required]: Starting selected output plugin PulseAudio Output.
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/Output/pulse_audio.so.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dcc7a170> buffering enabled for /home/user/.config/audacious/playlists/order
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dcc7a170> open (mode r) /home/user/.config/audacious/playlists/order
INFO ../audacious-4.4.2/src/libaudcore/playlist-files.cc:49 [playlist_load]: Loading playlist file:///home/user/.config/audacious/playlists/1000.audpl.
INFO ../audacious-4.4.2/src/libaudcore/playlist-files.cc:62 [playlist_load]: Trying playlist plugin Audacious Playlists (audpl).
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/Container/audpl.so.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dcc7b4e0> buffering enabled for file:///home/user/.config/audacious/playlists/1000.audpl
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dcc7b4e0> open (mode r) file:///home/user/.config/audacious/playlists/1000.audpl
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:188 [start_plugins]: Starting Audio CD Menu Items.
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/General/cd-menu-items.so.
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:188 [start_plugins]: Starting MPRIS 2 Server.
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/General/mpris2.so.
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:133 [start_required]: Starting selected interface plugin Winamp Classic Interface.
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/General/skins-qt.so.
INFO ../audacious-4.4.2/src/libaudcore/interface.cc:70 [interface_load]: Loading Winamp Classic Interface.
Detected locale "LC_CTYPE=C;LC_NUMERIC=sv_SE.UTF-8;LC_TIME=sv_SE.UTF-8;LC_COLLATE=C;LC_MONETARY=sv_SE.UTF-8;LC_MESSAGES=C;LC_PAPER=sv_SE.UTF-8;LC_NAME=sv_SE.UTF-8;LC_ADDRESS=sv_SE.UTF-8;LC_TELEPHONE=sv_SE.UTF-8;LC_MEASUREMENT=sv_SE.UTF-8;LC_IDENTIFICATION=sv_SE.UTF-8" with character encoding "ANSI_X3.4-1968", which is not UTF-8.
Qt depends on a UTF-8 locale, and has switched to "C.UTF-8" instead.
If this causes problems, reconfigure your locale. See the locale(1) manual
for more information.

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:167:17: expected ':'

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:292:11: Invalid name of pseudo-class

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:315:23: The :focused pseudo-class is deprecated. Use :focus instead.

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:401:20: 'max-height' is not a valid property name

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:402:19: 'max-width' is not a valid property name

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:403:16: 'height' is not a valid property name

(audacious:18164): Gtk-WARNING **: 16:17:29.543: Theme parsing error: widgets.css:404:15: 'width' is not a valid property name

(audacious:18164): Gtk-WARNING **: 16:17:29.544: Theme parsing error: widgets.css:850:23: 'share' is not a valid color name
QApplication: invalid style override 'gtk' passed, ignoring it.
	Available styles: Windows, Fusion
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dce79a80> buffering enabled for /tmp/audacious.PH2DW2/PLEDIT.TXT
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dce79a80> open (mode r) /tmp/audacious.PH2DW2/PLEDIT.TXT
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dce82630> buffering enabled for /tmp/audacious.PH2DW2/VISCOLOR.TXT
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dce82630> open (mode r) /tmp/audacious.PH2DW2/VISCOLOR.TXT
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x6294dce727a0> buffering enabled for /tmp/audacious.PH2DW2/REGION.TXT
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x6294dce727a0> open (mode r) /tmp/audacious.PH2DW2/REGION.TXT
INFO ../audacious-4.4.2/src/libaudcore/probe.cc:89 [aud_file_find_decoder]: Probing file:///mnt/sidecar/asl/%5BASL%5D%20_Various_Artist_EXIT%20TRANCE%20PRESENTS%20SPEED%20ANIME%20TRANCE%20%20BEST%202_%5Bw_Scans%5D_%5BFLAC%5D/07Hemisphere.flac.
INFO ../audacious-4.4.2/src/libaudcore/probe.cc:115 [aud_file_find_decoder]: Matched FLAC Decoder by extension.
INFO ../audacious-4.4.2/src/libaudcore/plugin-load.cc:64 [plugin_load]: Loading plugin: /usr/lib/audacious/Input/flacng.so.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x766f7c00a9a0> buffering enabled for file:///mnt/sidecar/asl/%5BASL%5D%20_Various_Artist_EXIT%20TRANCE%20PRESENTS%20SPEED%20ANIME%20TRANCE%20%20BEST%202_%5Bw_Scans%5D_%5BFLAC%5D/07Hemisphere.flac
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x766f7c00a9a0> open (mode r) file:///mnt/sidecar/asl/%5BASL%5D%20_Various_Artist_EXIT%20TRANCE%20PRESENTS%20SPEED%20ANIME%20TRANCE%20%20BEST%202_%5Bw_Scans%5D_%5BFLAC%5D/07Hemisphere.flac
INFO ../audacious-4.4.2/src/libaudcore/output.cc:270 [setup_output]: Setup output, format 0, 2 channels, 44100 Hz.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:50 [release_buffer]: <0x766f7c00a9a0> buffering disabled for file:///mnt/sidecar/asl/%5BASL%5D%20_Various_Artist_EXIT%20TRANCE%20PRESENTS%20SPEED%20ANIME%20TRANCE%20%20BEST%202_%5Bw_Scans%5D_%5BFLAC%5D/07Hemisphere.flac
INFO ../audacious-4.4.2/src/libaudcore/interface.cc:87 [interface_unload]: Unloading Winamp Classic Interface.
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:233 [stop_plugins]: Shutting down Audio CD Menu Items.
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:233 [stop_plugins]: Shutting down MPRIS 2 Server.
INFO ../audacious-4.4.2/src/libaudcore/probe-buffer.cc:30 [ProbeBuffer]: <0x766f900189d0> buffering enabled for /home/user/.config/audacious/playlists/order
INFO ../audacious-4.4.2/src/libaudcore/vfs.cc:105 [VFSFile]: <0x766f900189d0> open (mode r) /home/user/.config/audacious/playlists/order
INFO ../audacious-4.4.2/src/libaudcore/plugin-init.cc:218 [stop_plugins]: Shutting down PulseAudio Output.
```

**Comment by radioactiveman** [comment]
This is not a bug in Audacious but Qt 6.8.0 so we can't fix it. Older Qt versions render the Terminus font correctly.

See also: https://bugreports.qt.io/browse/QTBUG-130586

Full sample based on the bug report:

```c++
// c++ -fPIC $(pkg-config --cflags --libs Qt5Widgets) -> Terminus label visible
// c++ -fPIC $(pkg-config --cflags --libs Qt6Widgets) -> Terminus label invisible

#include <QApplication>
#include <QLabel>
#include <QVBoxLayout>

int main(int argc, char **argv)
{
    QApplication app(argc, argv);
    auto unstyled = new QLabel("Not Terminus!");
    auto styled = new QLabel("Terminus!");
    styled->setStyleSheet("QLabel { font: 12pt Terminus }");
    QWidget window;
    auto layout = new QVBoxLayout(&window);
    layout->addWidget(unstyled);
    layout->addWidget(styled);
    window.show();
    return app.exec();
}
```

**Comment by radioactiveman** [comment]
Fixed now with Qt 6.8.1, confirmed on Arch Linux.
